inotify.go

```diff
@@ -245,6 +245,15 @@ func (w *Watcher) readEvents() {
 
 			mask := uint32(raw.Mask)
 			nameLen := uint32(raw.Len)
+
+			if mask&unix.IN_Q_OVERFLOW != 0 {
+				select {
+				case w.Errors <- errors.New("inotify queue overflow"):
```

Comment:
This needs to be a well-defined error otherwise it's not easy for client code to determine that this condition has occurred without inspecting the error string (gasp!).

How about something like:

```go
  var ErrEventOverflow = errors.New("fsnotify queue overflow")
```

(this needs to be placed in a platform-independent file)

so that the client code than check `if err == fsnotify.ErrEventOverflow {`